dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/settings/LazyLoaded.java

```diff
@@ -0,0 +1,20 @@
+package edu.harvard.iq.dataverse.settings;
+
+import java.util.function.Supplier;
+
+class LazyLoaded<T> {
```

Comment:
More grammatically correct would be `LazilyLoaded` or `LazyLoad` but I'm not insisting on changing this.